classes/classes/Creature.as

```diff
@@ -3501,7 +3501,7 @@ public class Creature extends Utils
 		
 		public function hornDescript():String
 		{
-			return Appearance.DEFAULT_HORNS_NAMES[horns.type] + " _horns";
+			return Horns.Types[horns.type].name + " horns";
```

Comment:
Instead of a lookup for an existing object like this, I would prefer the horns object to set its own properties based on its type when the type is set.